runtime/src/accounts_index.rs

```diff
@@ -32,9 +32,9 @@ pub struct AccountMapEntryInner<T> {
 
 #[self_referencing]
 pub struct ReadAccountMapEntry<T: 'static> {
-    pub owned_entry: AccountMapEntry<T>,
+    owned_entry: AccountMapEntry<T>,
```

Comment:
@joshua-maros I was wondering why this wasn't working when I removed the `pub` 😄